src/main/kotlin/com/tealcube/minecraft/bukkit/mythicdrops/items/factories/MythicCustomItemFactory.kt

```diff
@@ -88,6 +90,14 @@ internal class MythicCustomItemFactory(
         }
         itemStack.itemFlags = customItem.itemFlags
         itemStack.setPersistentDataString(mythicDropsCustomItem, customItem.name)
+
+        if (customItem.rgb.isNotEmpty() && itemStack.itemMeta is LeatherArmorMeta) {
+            val meta = itemStack.itemMeta as LeatherArmorMeta
+            val colors = customItem.rgb.split(",")
+            meta.setColor(Color.fromRGB(colors[0].toInt(), colors[1].toInt(), colors[2].toInt()))
```

Comment:
Continuing from other comments:
```suggestion
            meta.setColor(customItem.rgb.toColor())
```